.github/workflows/readme.yml

```diff
@@ -0,0 +1,37 @@
+name: Translate README
+
+on:
+  push:
+    branches:
+      - main
+      - master
```

Comment:
Let's remove the reference to master here as we only have a "main" branch and want to be inclusive

Reply:
Yes, we can. I am making the required changes. Thank you for the feedback.